scripts/install_third_party_libs.py

```diff
@@ -125,9 +125,19 @@ def pip_install(package, version, install_path):
 
     # The call to python -m is used to ensure that Python and Pip versions are
     # compatible.
-    subprocess.check_call([
-        sys.executable, '-m', 'pip', 'install', '%s==%s' % (package, version),
-        '--target', install_path])
+    try:
+        fnull = python_utils.open_file(os.devnull, 'w')
+        subprocess.check_call([
+            sys.executable, '-m', 'pip', 'install',
+            '%s==%s' % (package, version),
+            '--target', install_path], stdout=fnull, stderr=subprocess.STDOUT)
+    except Exception as ex:
```

Comment:
Can you try to use more specific exception here?

Reply:
I tried to get an exception that describes the error message (can't combine user with prefix) but I could not, This prints an error code. I don't know how to do anything else, can you please help me with that.
Thanks for reviewing!